exampleApp/src/user/userAuthenticator.ts

```diff
@@ -0,0 +1,18 @@
+import { User } from './user.entity'
+import { TypeOrmModule, getRepositoryToken } from '@nestjs/typeorm'
+import { Repository } from 'typeorm'
+
+export const UserAuthenticator = {
```

Comment:
Could we replace all these `Authenticator` with `CredentialValidator`? That's more explicit I think